File: src/main/java/com/iflat/pam/bean/PartyGroup.java
package com.iflat.pam.bean;

/**
 * Created by tyriv on 2016/7/29.
 */
public class PartyGroup {

    private String id;
    private String pbName;
    private String name;
    private String leader;
    private int memberNum;
    private String group;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getPbName() {
        return pbName;
    }

    public void setPbName(String pbName) {
        this.pbName = pbName;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLeader() {
        return leader;
    }

    public void setLeader(String leader) {
        this.leader = leader;
    }

    public int getMemberNum() {
        return memberNum;
    }

    public void setMemberNum(int memberNum) {
        this.memberNum = memberNum;
    }

    public String getGroup() {
        return group;
    }

    public void setGroup(String group) {
        this.group = group;
    }
}

File: web/app/model/pam/PartyGroup.js
Ext.define('iFlat.model.pam.PartyGroup', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'partyGroup.id', mapping: 'id', type: 'string'},
        {name: 'partyGroup.pbName', mapping: 'pbName', type: 'string'},
        {name: 'partyGroup.name', mapping: 'name', type: 'string'},
        {name: 'partyGroup.leader', mapping: 'leader', type: 'string'},
        {name: 'partyGroup.memberNum', mapping: 'memberNum', type: 'number'},
        {name: 'partyGroup.group', mapping: 'group', type: 'string'},
    ]
});